frontend/src/static/css/v_authorship.scss

```diff
@@ -19,7 +19,7 @@
 
       .radio-button--checkbox {
         float: left;
-        margin: 0 2.0rem 0 -3rem;
+        margin: 0 2.0rem 0 0;
```

Comment:
can combine `radio-button--checkbox` and `radio-button--search` to `radio-button`

Reply:
Doing so will have to change other things like the upper margin of the search bar, etc. Would like to minimise changes if possible as it is a bug fix not refactoring.